src/Sentry/Internal/PlatformIntegration.uwp.cs

```diff
@@ -0,0 +1,54 @@
+#if WINDOWS_UWP
+using System;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Security;
+using Sentry.Protocol;
+using Windows.UI.Xaml;
+
+namespace Sentry.Internal
+{
+    internal class PlatformIntegration : IInternalSdkIntegration
+    {
+        private SentryOptions _options;
+        private IHub? _hub;
+
+        public void Register(IHub hub, SentryOptions options)
+        {
+            _options = options;
+            _hub = hub;
+            //Sentry default Logger doesn't work with UWP.
+            options.AddEventProcessor(new PlatformEventProcessor(options));
+            var uwpApplication = Application.Current;
```

Comment:
What kind of hooks can we get here?
I assume OnSuspending and all that? Maybe we use to persist state like Scope or to flush out events. Or for auto breadcrumb for UI navigating or some system events

Reply:
Added a hook for registering when the app is going to background or foreground.